app/models/agendaitem.js

```diff
@@ -68,19 +68,17 @@ export default ModelWithModifier.extend({
     return getPropertyLength(this, 'documentContainers');
   }),
 
-  linkedDocumentContainersLength: computed('linkedDocumentContainers', function() {
-    return getPropertyLength(this, 'linkedDocumentContainers');
-  }),
 
   documentContainers: computed('pieces.@each.name', function() {
     return PromiseArray.create({
       promise: this.get('pieces').then((pieces) => {
         if (pieces && pieces.get('length') > 0) {
-          const pieceIds = pieces.mapBy('id').join(',');
           return this.store.query('document-container', {
             filter: {
               pieces: {
-                id: pieceIds,
+                agendaitem: {
```

Comment:
Op het eerste zicht lijkt hier een vergissing in het spel. Een `document-container` heeft geen koppeling met `agendaitem`?